src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -88,6 +88,7 @@ public void setTestVerbosity(){
      */
 
     public static final String NA12878_chr17_1k_BAM = publicTestDir + "NA12878.chr17_69k_70k.dictFix.bam";
+    public static final String NA12878_chr17_1k_CRAM = publicTestDir + "NA12878.chr17_69k_70k.dictFix.cram";
```

Comment:
Does this have a .crai index? If it's declared in `BaseTest` a lot of tests will end up using it, so it should be usable in all cases.